### PR TITLE
Chap 4 tweak

### PR DESCRIPTION
**Pull request by jkrvivian**
* trading model
* game theory

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -63,7 +63,7 @@ \subsection{Participants}
 There are four major roles in the decentralized data marketplace (Figure \ref{fig:system_design}).
 
 \subsubsection{Registrar}
-Registrars have access to create and maintain Registration Contracts, a lookup table of participants in decentralized data marketplace. 
+Registrars have access to create and maintain registration contracts, a lookup table of participants in decentralized data marketplace.
```
Comment (jserv) [inline paper.tex]: I would like to keep the upper case form " Registration Contract".

File: paper.tex [hunk 2]
```diff
@@ -115,12 +115,12 @@ \subsection{Launching and Searching Products}
     \label{fig:key_certification_exchange}
 \end{figure}
 
-The contract address and product description will be stored in a file which is uploaded to IPFS. Consumers can search the desired product by keywords or tags. The consumer then evaluates the product and initiates the trading with the data provider if the consumer is interested in subscribing to the data.
+The contract address and product description will be stored in a file which is uploaded to IPFS. Consumers can search the desired product by keywords or tags and then launch a trading process for interested products.
 
 \subsection{Trading}
-Once a consumer who wants to subscribe certain streaming data that is generated  pays a subscription fee to the product contract, he/she is added to the consumers list automatically by the smart contract.
+Once a consumer subscribe to certain streaming data that is generated pays a subscription fee to the purchase contract, he/she is added to the consumers list automatically by the smart contract.
```
Comment (jserv) [inline paper.tex]: Should be "subscribes"

File: paper.tex (hunk 1 above)
Comment (jkrvivian) [reply paper.tex]: What about "product contract" and "purchase contract"?

Comment (jserv) [reply paper.tex]: I defer to @twzjwang .

**Pull request merged**